projects/plugins/jetpack/modules/search/customberg/components/sidebar/index.jsx

```diff
@@ -30,13 +31,14 @@ export default function Sidebar( props ) {
 				tabindex="-1"
 			>
 				<Tabs enabledSidebarName={ enabledSidebarName } enableSidebar={ enableSidebar } />
+				<SaveButton />
 				<Button
-					aria-label={ __( 'Close settings', 'jetpack' ) }
+					aria-label={ __( 'Show preview', 'jetpack' ) }
```

Comment:
Could we use the `label` attribute, which would fill `aria-label` and Tootip as well - I guess it would be nice to have some text for buttons with just an icon.

Reply:
The `label` attribute doesn't actually render the text within the button but instead applies it to `aria-label` and to the tooltip body. Tooltips are often ineffective in mobile contexts, so it won't be good for us here (and I don't think you were proposing that we adopt tooltips here, at any rate).

I'd like to push back against adding verbose labels like `Show preview` and `Show settings` in these buttons for mobile. For horizontally constrained viewports, doing so makes our layout break in undesirable ways:

|Before|After|
|---|---|
|<img width="251" alt="Screen Shot 2021-10-04 at 5 35 39 PM" src="https://user-images.githubusercontent.com/4044428/135938285-b97799ec-d20f-41b9-824d-3031dca632e5.png">|<img width="251" alt="Screen Shot 2021-10-04 at 5 32 09 PM" src="https://user-images.githubusercontent.com/4044428/135938046-f701fe5e-8e45-496e-aeb9-a97a78a9359f.png">|
|<img width="251" alt="Screen Shot 2021-10-04 at 5 35 36 PM" src="https://user-images.githubusercontent.com/4044428/135938311-c1a5c177-2bea-4220-9516-cfb6c816d0d7.png">|<img width="251" alt="Screen Shot 2021-10-04 at 5 32 04 PM" src="https://user-images.githubusercontent.com/4044428/135938048-7d243666-f858-48a5-ac72-eb643bd2daaa.png">|

Reply:
ha I thought the text would be hovering tootip. All good mate, thanks for trying it out.